drafts/one-percent-for-open-source.rst

```diff
@@ -0,0 +1,89 @@
+One Percent for Open Source
+===========================
+
+The open source ecosystem is the most valuable part of the software industry today.
+All of the companies and technologies that we build today are enabled by open source software.
+From the programming languages to the web frameworks,
+the operating system to the cryptography,
+all of it is open source.
+
+Open source has a `funding problem`_.
+There are many `warning signs`_ that the future is not as bright as the past.
+
+Open source is maintained for "free" by people all over the world,
+enabled by the internet (which is also built on open source).
+Many people doing this work are burning out,
+giving up,
+or just never standing in the first place.
+We need to enable the sustainability of open source in order to protect this commons that we currently have,
+and help it grow into the future.
+
+Funding Open Source
+-------------------
+
+I'd like to propose an idea:
+
+    A program called **1% for open source**: A program where Venture Capitalist's and startup companies agree to invest 1% of their funding round into the open source software that they use. 
+
+I believe that this is a win/win/win for all involved:
+
+* **Venture Capalists** invest in the open source ecosystem that makes companies *much cheaper to build*. It's also *really good marketing* for the VC firms, to show they are investing in the common tools we all rely on.
+* **Startups** invest in the tools that they use, to *ensure continued support and maintanance*, and remove risk from their future. It will also make *hiring developers* much easier for the companies, because developers love companies that invest in open source.
+* **Open source projects** are able to *fund project management and maintanance* that is required to make them sustainable for the long term. This continues the *force multiplier* that open source has given to the software industry as a whole.
+
+Building a standard model for investing in open source,
+and a point in time where we do it,
+will help create more investment in our existing infrastructure.
+New software projects will also be created,
+because now there is a more workable sustainability model.
+
+Why Venture Capital payments?
+-----------------------------
+
+Everyone involved in the above transaction gets values from open source.
+I believe that everyone also gets value from the 1% for open source program,
+through marketing and better software over time.
+
+Having a transaction that is already happening,
+with all parties benefiting,
+is a natural place to redistribute some of the money that is changing hands.
```

Comment:
I agree and would avoid using the word "redistribute". It might be even worse than "tax".

Instead, what I think you're saying is that a venture capital investment is a natural place to make sure that the core infrastructure that the fledgling company relies upon is actively maintained and improved.